testing/lshell/APKBUILD

```diff
@@ -2,39 +2,35 @@
 # Maintainer: Fabian Affolter <fabian@affolter-engineering.ch>
 pkgname=lshell
 pkgver=0.9.18
-pkgrel=1
+pkgrel=2
 pkgdesc="Python-based limited Shell"
 url="http://ghantoos.org/limited-shell-lshell"
 arch="noarch"
 license="GPL3+"
 depends="python2"
-depends_dev=""
+options="!check"
```

Comment:
the testsuite requires `lshell` to be installed - so I bundled the tests rather than run the testsuite against the previous version of `lshell` (I was going to run the testsuite on my servers - but since making this `PR` I have used `rshell` instead)

if you think it's better to run the testsuite against the previous version of `lshell` I will revise & rebase.